packages/modular-scripts/craco.config.js

```diff
@@ -80,6 +80,15 @@ module.exports = {
 
       return {
         ...jestConfig,
+        reporters: [
+          'default',
+          [
+            'jest-junit',
```

Comment:
I agree this feature would valuable to include, but thinking about whether it should happen by default. How would you feel about putting this behind a flag? Say, `--coverage-junit` or something.